tests/components/ring/test_sensor.py

```diff
@@ -57,6 +63,24 @@ async def test_sensor(hass: HomeAssistant, requests_mock: requests_mock.Mocker)
     assert front_door_wifi_signal_strength_state.state == "-58"
 
 
+async def test_history(
+    hass: HomeAssistant,
+    freezer: FrozenDateTimeFactory,
+    requests_mock: requests_mock.Mocker,
+) -> None:
+    """Test history derived sensors."""
+    await setup_platform(hass, Platform.SENSOR)
+    freezer.tick(120)
+    await setup_platform(hass, Platform.SENSOR)
+    await hass.async_block_till_done()
```

Comment:
We shouldn't need to call `setup_platform` twice and tick for 120.  Try this:
```suggestion
    hass.config.set_time_zone("UTC")
    freezer.move_to("2021-01-09 12:00:00+00:00")
    await setup_platform(hass, Platform.SENSOR)
    await hass.async_block_till_done()
    freezer.tick(SCAN_INTERVAL)
    async_fire_time_changed(hass)
    await hass.async_block_till_done()
```